lib/minio_runner/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module MinioRunner
-  VERSION = "0.1.3"
+  VERSION = "1.0.0"
```

Comment:
Major version bump because we are changing minimum Ruby version which is backwards incompatible. That being said, I'm quite sure we are the only consumer of this gem...